### PR TITLE
PR for issue 142

### PR DESCRIPTION
**Pull request by jwang-101**
Work-in-progress on Issue https://github.com/oss-slu/lrda_website/issues/142
Attempts to add a popup after the login screen that greets a user.
This was added to ease new viewers into Where's Religion.
On the Home page, which is what is brought up after the login, intro.js was used. Intro.js was imported, then added with the useEffect method to generate the pop up message.

### CODE REVIEW

**Comment by jwang-101** [comment]
1. Work-in-progress on Issue #142 
2. Attempts to add a popup after the login screen that greets a user.
3. This was added to ease new viewers into Where's Religion.
4. On the Home page, which is what is brought up after the login, intro.js was used. Intro.js was imported, then added with the useEffect method to generate the pop up message.

File: app/lib/components/search_bar_ui.tsx [hunk 1]
```diff
@@ -1,18 +1,43 @@
-import React from 'react';
+import React, {useRef, useEffect} from "react";
```
Comment (amychen108) [inline app/lib/components/search_bar_ui.tsx:1]: This file does not need to be changed, all the additional code should be removed.

File: pnpm-lock.yaml
Comment (kate-holdener) [inline pnpm-lock.yaml:1]: @jwang-101 @yashb196 @BaloneyBoy97 Are these changes intentional/necessary? It's a lot of changes.

**Comment by kate-holdener** [comment]
> Work-in-progress on Issue #142 Attempts to add a popup after the login screen that greets a user. This was added to ease new viewers into Where's Religion. On the Home page, which is what is brought up after the login, intro.js was used. Intro.js was imported, then added with the useEffect method to generate the pop up message.

Please provide required information in your PR description (as discussed in class, see class slides for details)

File: pnpm-lock.yaml
Comment (BaloneyBoy97) [reply pnpm-lock.yaml:1]: this is a auto update from the lock file
